08-Check-ODAP-Tables.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+import sys
+from shared_python.Args import Args
+from shared_python.Sql import Sql
+
+if __name__ == "__main__":
+    """Perform various audits to ensure data quality on the ODAP tables prior to
+    export."""
+    args_obj = Args()
+    args = args_obj.args_for_08()
+
+    log = args_obj.logger_with_filename()
+    sql = Sql(args, log)
+    db: str = args.output_database
+    found_error = False
+
+    log.info(f"Performing quality checks on {db}")
+    sql.execute(f"USE {db}")
+
+    ##
+    ## Step O: Check for excessive tags
+    ##
+
+    # The archive will error if a work has more than 75 total tags.
```

Comment:
```suggestion
    # The Archive will error if a work has more than 75 total tags.
```
(capitalise instances of "Archive" in the sense of AO3 to distinguish it from the archive we're processing)